### PR TITLE
Fixed Several Typos

### PR DESCRIPTION
**Pull request by rajasekharporeddy**
Fixed Typos in several doc files.

### CODE REVIEW

File: docs/jep/4410-omnistaging.md [hunk 1]
```diff
@@ -323,7 +323,7 @@ print(init())  # 0.48648298  !!
 ```
 
 That last call has repeated randomness but no hard error, because we aren't
-re-executing the Python. But if we look at `key`, we see an escaped tracer _when
+re-executing Python. But if we look at `key`, we see an escaped tracer _when
```
Comment (jakevdp) [inline docs/jep/4410-omnistaging.md:326]: Please revert this change, I think the original was more correct.

File: docs/jep/9407-type-promotion.ipynb [hunk 2]
```diff
@@ -296,7 +296,7 @@
...
...aL1kyRLExcXhnXfeweuvvw6bzYbRo0cjJyeH2TFoJ27SkRb5kJubC1mWIUkSEhISkJ+fj+HDhzM9BuXx/SmK0mOhE8T6O8zNzYXJZIIsy0hJScFLL72EBQsWMD0GwHfsDbnTMuGfz+dDZWUlV8urkiRh+PDh3N7wR9ijPBZPRUUFxo4di8WLF3db6ARR7EOPVngIlzo+os4TXudNtMFrPvA671BITU3FnTt3EBUVdd/P8fod8jpvgPbhIZwKbhLJE6vVynX/m7BHeSymBxU7AMU+HKjgIdyy2+3hnsJD4W2+JDR4ywve5qtnvH2XvM23Kyp4CLccDgc3N9BJkgSHwxHuaRAdojw2Lop9aPHxTRPSDbvdzs3JwmQycX91RLRBeWxcFPvQ4uObJqQbkiQhLi5O9z1lSZIQHx+v+3mS8KA8Ni6KfWhRwUO4FhsbG+4p9IrT6Qz3FIiOUR4bF8U+dKjgIVwzmUxwOp26vfKQJAlOpxMmkyncUyE6RnlsXBT70KGCh3DP5XLp9pfRZDLB5XKFexqEA5THxkWxDw0qeAj3ZFlGYmKi7q6QJElCYmIiNzclkvCiPDYuin1oiPFvQQzPbrfralk4uAzM+1MNJLQoj42LYq89KniIMFwuFyIjI8N+wpAkCZGRkcIsA5PQojw2Loq9tqjgIcKQZRlJSUlh3f48uF18UlKSMMvAJLQoj42LYq8tels6EY6iKKiurobX6w3pm4iDV0VJSUm6vQGR8IPy2Lgo9tqggocIKRAIoLa2Fo2NjSE5YQT73S6XS7irIhI+lMfGRbFnjwoeIjSPx4OamhooiqLJSUOSJJhMJiQmJgp1cx/RF8pj46LYs0MFDxFexyslAExOGsH+uuhXREQ/KI+Ni2LPBhU8xDAURUFjYyPq6+v7fLUUvBqKj48XZvdRwhfKY+Oi2PcPFTzEcFRVhcfjgdvthsfjgc/nA4Bun4oI/npYrVbY7XY4HA7Y7fawPzZKCOWxcVHs+4YKHmJ4qqrC7/fD7/cjEAhAVVVIkgRZlmGxWGCxWAx5ciB8oTw2Lop971DBQwghhBDhiX+XEiGEEEIMjwoeQgghhAiPCh5CCCGECI8KHkIIIYQIjwoeQgghhAiPCh5CCCGECI8KHkIIIYQIjwoeQgghhAiPCh5CCCGECO//AcN1Ixa6KgO0AAAAAElFTkSuQmCC\n",
```
Comment (jakevdp) [inline docs/jep/9407-type-promotion.ipynb:299]: What changes did you make to these images? It's difficult to tell from the code diff. Perhaps revert, since this doesn't have anything to do with fixing typos?

**Review by jakevdp — CHANGES_REQUESTED** [review]
Thanks! A few comments inline.

File: docs/jax-101/06-parallelism.md [hunk 3]
```diff
@@ -275,7 +275,7 @@ Here's how `update()` works:
...
 
-We want to spread the `batch` dimension across all available devices. To do that, we add a new axis using `pmap`. The arguments to the decorated `update()` thus need to have shape `[num_devices, batch_per_device, ...]`. So, to call the new `update()`, we'll need to reshape data batches so that what used to be `batch` is reshaped to `[num_devices, batch_per_device]`. That's what `split()` does below. Additionally, we'll need to replicate our model parameters, adding the `num_devices` axis. This reshaping is how a pmapped function knows which devices to send which data.
+We want to spread the `batch` dimension across all available devices. To do that, we add a new axis using `pmap`. The arguments to the decorated `update()` thus need to have the shape `[num_devices, batch_per_device, ...]`. So, to call the new `update()`, we'll need to reshape data batches so that what used to be `batch` is reshaped to `[num_devices, batch_per_device]`. That's what `split()` does below. Additionally, we'll need to replicate our model parameters, adding the `num_devices` axis. This reshaping is how a pmapped function knows which devices to send which data.
```
Comment (jakevdp) [inline docs/jax-101/06-parallelism.md:278]: This reads better to me without the "the"

**Review by jakevdp — CHANGES_REQUESTED** [review]
Thanks for the updates! One more comment, and then please squash all changes into a single commit so we can keep our commit history clean (see https://jax.readthedocs.io/en/latest/contributing.html#single-change-commits-and-pull-requests) thanks!

**Comment by rajasekharporeddy** [comment]
Hi @jakevdp 

Thanks for reviewing. Squashed all commits into single commit. Please let me know if any changes needs to be done.

**Review by jakevdp — APPROVED** [review]
Thanks!

**Comment by jakevdp** [comment]
It looks like some of the notebook sources are unsynced; see https://jax.readthedocs.io/en/latest/developer.html#syncing-notebooks for how to fix this.

**Review by jakevdp — APPROVED** [review]
Thanks!
